v23/vdlroot/vdltool/config.vdl

```diff
@@ -238,3 +240,7 @@ type SwiftConfig struct {
 	// in Swift vdl package.
 	WireToNativeTypes map[string]string
 }
+
+// PythonConfig specifics python specific configuration for this package.
```

Comment:
s/specifics/specifies/

Reply:
I removed the python config since it doesn't really belong in this revision in any case.

Reply:
I still see the Python on line 23 and 32. The Travis CI also indicates that v.io/x/ref/lib/vdl/build is not happy. I was got the same error using the vdl-python branch from cosnicolaou/core.